<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>505</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea &gt; QWidget &gt; QWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(55, 71, 79, 255), stop:0.5 rgba(38, 50, 56, 255), stop:1 rgba(34, 42, 48, 255));
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>510</width>
        <height>485</height>
       </rect>
      </property>
      <widget class="RotatedWidget" name="destinationTargetPreview">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>187</y>
         <width>241</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">EntityWidget {
    
	border-image: url(:/logoUP.png);
}</string>
       </property>
       <property name="opacity">
        <double>0.300000000000000</double>
       </property>
       <property name="blending">
        <enum>E::Overlay</enum>
       </property>
       <property name="respondable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="RotatedWidget" name="target">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>241</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">EntityWidget {
    
	border-image: url(:/logoUP.png);
}</string>
       </property>
       <property name="respondable">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::DontWrapRows</enum>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="posLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>geometry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="curr_x">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="curr_y">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="curr_w">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>260</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="curr_h">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>260</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="posLabel_2">
       <property name="text">
        <string>new geometry</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="dest_x">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QSpinBox" name="dest_y">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="dest_w">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="dest_h">
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="btn_go">
       <property name="text">
        <string>go</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="EasingCurveComboBox" name="easingcurvecombobox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="posLabel_3">
       <property name="text">
        <string>easing curve</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="input_duration">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>2500</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="posLabel_4">
       <property name="text">
        <string>duration(msec)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="posLabel_5">
       <property name="text">
        <string>direction</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="bttn_forward">
         <property name="text">
          <string>forward</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bttn_backward">
         <property name="text">
          <string>backward</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="posLabel_6">
       <property name="text">
        <string>rotation</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="input_rotation">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EasingCurveComboBox</class>
   <extends>QComboBox</extends>
   <header>plugins.h</header>
  </customwidget>
  <customwidget>
   <class>RotatedWidget</class>
   <extends>EntityWidget</extends>
   <header>animation/rotatedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../poker_client/data/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
